actioncable/lib/action_cable/channel/streams_handlers/base.rb

```diff
@@ -0,0 +1,23 @@
+module ActionCable
+  module Channel
+    module StreamsHandlers
+      class Base # :nodoc:
+        attr_reader :channel, :identifier
+
+        delegate :connection, :identifier, to: :channel
+
+        def initialize(channel, **options)
+          @channel = channel
+        end
+
+        def call(message)
+          # We can skip message decoding and generate payload in a more simple way
+          connection.transmit(
+            "{\"identifier\":#{identifier.to_json},\"message\":#{message}}",
```

Comment:
No, we should call `to_json` (which adds surrounding quotes and escapes existing quotes), 'cause `identifier` is a JSON string itself.